lib/utils/index.js

```diff
@@ -1015,6 +1015,12 @@ module.exports = {
    * @returns {VElement | null} the element of `<script setup>`
    */
   getScriptSetupElement,
+  /**
+   * Gets the element of `<script>` that is NOT setup
+   * @param {RuleContext} context The ESLint rule context object.
+   * @returns {VElement | null} the element of `<script>`
+   */
+  getScriptElement,
```

Comment:
Are the changes in this file really necessary? Other similar rules work with the existing utils.